### ISSUE TITLE
Jump to library after update to v2.2.5

### ISSUE THREAD

**Issue by Marcaay** [jmooremcc/plugin.program.super.favourites#5]
Hi John,

not sure how to contact you again after you closed issue #4
This fixed the problem with opening the series links. Many thanks for your good work.

I would like to inform that the previous versions were staying within the folder from which the links are started. With latest release kodi jumps to the video library view, if you are starting a link and stopping it then, out of a SF folder.
Additionally I detected that I can't enter a SF folder afterwards. I need to restart Kodi again. 

If possible to handle the series links correct and stays within current used SF folder it would be perfect.

If I can support with testing, please let me know.

Is it possible to download previous versions from your github page? I can't find it. 

Marc

**Comment by jmooremcc** [comment]
@Marcaay 
Hi Marc,
I wish there was a way for you to do a video that shows me the problem you are describing.
In the meantime, the repository has zip files for all the past releases you can download if you need to.

John

**Comment by jmooremcc** [comment]
Hi Mark,
This might not be the same as what you did but I have a folder that has a link to Crackle's horror folder.
I clicked on the Crackle folder and it displayed a list of movies.
I partially played two movies and after I stopped the second movie, I hit the back button which took me back to the original SF folder that I launched from. Hitting the backspace key again took me up to the next level.

Is this different from the procedure you followed?

John
